project4/styles.css

```diff
@@ -0,0 +1,301 @@
+:root {
+    --color-black: #000000;
+    --color-pink: #c6177a;
+    --color-white: whitesmoke;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: var(--color-black);
+    color: whitesmoke;
+    font-family: montserrat-alternates, sans-serif;
+    margin: 0;
+}
+
+header img,
+header div {
+    display: inline-block;
+}
+
+header div {
+    text-align: center;
+}
+
+header h3 {
+    margin: 0;
+}
+
+h1,
+h2,
+h3 {
+    color: var(--color-pink);
+}
+
+h1,
+h2 {
+    font-family: bigsmalls-bold, sans-serif;
+}
+
+h3,
+h4 {
+    font-family: bigsmalls-regular, sans-serif;
+
+}
+
+h1 {
+    font-size: 15px;
+    line-height: 1.5;
+    margin: 0;
+}
+
+h2 {
+    font-size: 50px;
+}
+
+p {
+    font-family: montserrat-alternates, sans-serif;
+}
+
+nav ul {
+    font-family: bigsmalls-regular, sans-serif;
+    list-style: none;
+    padding-left: 0;
+    text-align: center;
+}
+
+nav li {
+    display: inline-block;
+    line-height: .8;
+    max-width: 75px;
+
+}
+
+nav li a {
+    color: var(--color-white);
+    font-size: 10px;
+    line-height: 0;
```

Comment:
`line-height: 0;` for anything that has text in it is always worrisome and will lead to troubles:
```suggestion
```

Also, if you make this element `display: block;`here, your hover effect will look much better:

**Before**
<img width="143" alt="before" src="https://user-images.githubusercontent.com/2653980/70740471-40e51700-1ce7-11ea-9525-93124aa0dec6.png">

**After**
<img width="187" alt="after" src="https://user-images.githubusercontent.com/2653980/70740478-43477100-1ce7-11ea-8924-56610d5903e0.png">